Examples/Physics_applications/laser_acceleration/inputs_1d

```diff
@@ -0,0 +1,66 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 1000
+amr.n_cell =  512
+amr.max_grid_size = 64   # maximum size of each AMReX box, used to decompose the domain
+amr.blocking_factor = 32 # minimum size of each AMReX box, used to decompose the domain
+geometry.coord_sys   = 0                  # 0: Cartesian
+geometry.prob_lo     =  -56.e-6    # physical domain
+geometry.prob_hi     =   12.e-6
+amr.max_level = 0 # Maximum level in hierarchy (1 might be unstable, >1 is not supported)
+
+#################################
+####### Boundary condition ######
+#################################
+boundary.field_lo = pec
+boundary.field_hi = pec
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.do_dive_cleaning = 0
+warpx.use_filter = 1
+warpx.cfl = 0.9 # if 1., the time step is set to its CFL limit
+warpx.do_moving_window = 1
+warpx.moving_window_dir = z # Only z is supported for the moment
+warpx.moving_window_v = 1.0 # units of speed of light
+
+# Order of particle shape factors
+algo.particle_shape = 3
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = electrons
+
+electrons.species_type = electron
+electrons.injection_style = "NUniformPerCell"
+electrons.num_particles_per_cell_each_dim = 1
```

Comment:
Maybe adding a few particles more makes sense for 1D:
```suggestion
electrons.num_particles_per_cell_each_dim = 10
```

Reply:
Agreed!